testdata/lineage/basic-multiversion.txtar

```diff
@@ -0,0 +1,337 @@
+# Basic schema evolution over time
```

Comment:
As we discussed, I thought about re-using the one defined at `testdata/internal/basic.cue`, but there are some subtle differences, so I'm not sure if it's better to keep them for separate or try to bring examples there and try to re-use it. So far, I decided to start with the simpler.

Reply:
i think it's fine to keep it simpler/separate - `testdata/internal/basic.cue` was intended to be part of a pure CUE corpus of tests, which is probably still worth having. (We could actually spit one out from the main corpus, then use e.g. `cue eval` and/or invoke the `thema` binary. That's also probably what we'll want to attach to unity - #121)

But for now, that doesn't exist, so making this part of the main corpus is 👍🏻